pkg/openshift/certgen/templates/master/tmp/ansible/azure-local-master-inventory.yml

```diff
@@ -52,9 +52,11 @@ localmaster:
     openshift_hosted_router_replicas: 1
     openshift_hosted_registry_replicas: 1
 
+    openshift_prometheus_state: "present"
 
     openshift_web_console_install: True
 
+    openshift_prometheus_image_prefix: "registry.reg-aws.openshift.com:443/openshift3/"
```

Comment:
shouldn't have any references to registry.reg-aws.openshift.com in this file.  One slipped in above on line 17 at some point too.  Please factor them out.  In theory should write TEMPIMAGEBASE but that won't work directly because it ends in ose-.  I guess IMAGE_BASE needs to be refactor into separate parts.